types/jss/css.d.ts

```diff
@@ -0,0 +1,134 @@
+// These CSS typings adapted from TypeStyle: https://github.com/typestyle/typestyle
+
+import { Observable } from './observable';
+import * as csstype from 'csstype';
+
+type Length = string | number;
+
+export type ObservableProperties<P> = {
+    [K in keyof P]: P[K] | Observable<P[K]>
+};
+
+export type CSSProperties =
+	& ObservableProperties<csstype.Properties<Length>>
+	& ObservableProperties<csstype.PropertiesHyphen<Length>>;
+
+export interface JssProps {
+    '@global'?: CSSProperties;
+    extend?: string;
+    composes?: string | string[];
+}
+
+export interface JssExpand {
+    animation:
+        | {
+              delay: CSSProperties['animationDelay'];
+              direction: CSSProperties['animationDirection'];
+              duration: CSSProperties['animationDuration'];
+              iterationCount: CSSProperties['animationIterationCount'];
+              name: CSSProperties['animationName'];
+              playState: CSSProperties['animationPlayState'];
+              timingFunction: CSSProperties['animationTimingFunction'];
+          }
+        | CSSProperties['animation'];
+    background:
+        | {
+              attachment: CSSProperties['backgroundAttachment'];
+              color: CSSProperties['backgroundColor'];
+              image: CSSProperties['backgroundImage'];
+              position: CSSProperties['backgroundPosition'] | number[]; // Can be written using array e.g. `[0 0]`
```

Comment:
Shouldn't this be `[number, number]` to constrain the array length?

Reply:
Sorry, but I'm not familiar with JSS so I just do as I'm told until someone else objects. 😸 I changed to `[csstype.Properties['backgroundPosition'], csstype.Properties['backgroundPosition']]` because I assume that `['center', 'center']` can be used as well?